app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -210,6 +211,14 @@ class SettingsActivity : DuckDuckGoActivity() {
             .launchIn(lifecycleScope)
     }
 
+    private fun updatePrivacyPro(isPrivacyProEnabled: Boolean) {
+        if (isPrivacyProEnabled) {
```

Comment:
NIT: i believe we should also hide the _old_ VPN entry, assuming we are not removing it before launch

Reply:
Good call, will do in the view model instead